Blog - Camel 3 is EOL (#1272)
apache/camel-website: content/blog/2024/12/camel3-eol/index.md

---
title: "Apache Camel 3 - End of Life"
date: 2024-12-23
authors: [davsclaus]
categories: ["Roadmap"]
preview: Apache Camel 3 is end of life
---

Apache Camel is officially end of life by end of 2024.

We have just [released](/blog/2024/12/RELEASE-3.22.3/) the last Apache Camel 3.22.3 release of Camel 3.x.

> There will NOT be any more releases and support for Camel 3 is end of life.

Reviewer: ```suggestion
There will be NO further releases, and support for Camel 3 has officially reached its end of life.
```

There will not be any new releases of Camel 3.x, and users are encouraged to start new development
on Camel 4, and plan to [migrate existing applications](/blog/2023/10/migrate4/) to Camel 4.x.

> We have done a total of 78 Camel 3.x releases, since Camel 3.0.0 was released (november 2019) 5 years ago.

Reviewer: ```suggestion
We have done a total of 78 Camel 3.x releases, since Camel 3.0.0 was released (November 2019) 5 years ago.
```
